### PR TITLE
style(signup): reintroduce retina images in the select field after select-row hack

### LINKED ISSUE #2120: Custom select box style is not using retina images

### PR DESCRIPTION
**Pull request by zaach**
It looks like a regression was introduced when we began using `.select-row` to workaround
`select` styling issues in FF30. We lost our retina images! No longer.

Fixes #2120.

Before:
screen shot 2015-02-19 at 1 26 11 pm

After:
screen shot 2015-02-19 at 2 12 08 pm

@vladikoff or @shane-tomlinson r?

### CODE REVIEW

File: app/styles/_general.scss [hunk 1]
```diff
@@ -537,15 +538,15 @@ label:focus ~ .input-help-focused {
...
+       cursor: default;
 
-     select {
-      color: $input-placeholder-color;
-     }
-  }
+       select {
+        color: $input-placeholder-color;
+       }
+    }
```
Comment (zaach) [inline app/styles/_general.scss]: unrelated formating fix.

File: app/styles/_media_queries.scss [hunk 2]
```diff
@@ -320,11 +320,17 @@ only screen and (orientation:landscape) and (min-width:481px) and (max-height:48
...
   }
 
-  .input-row select {
+  .input-row select,
+  .select-row,
+  .select-row.disabled:hover {
     background-image: image-url('ddarrow_inactive@2x.png');
   }
 
-  .input-row select:focus {
```
Comment (shane-tomlinson) [inline app/styles/_media_queries.scss]: We only have 3 "select" elements, all inside of "select-row"s. I believe we can safely remove the ".input-row select"s.

**Comment by shane-tomlinson** [comment]
> We only have 3 "select" elements, all inside of "select-row"s. I believe we can safely remove the ".input-row select"s.

Opened #2126 to fix this.
